Drive/rover_drive_system.hpp

```diff
@@ -81,8 +85,8 @@ class RoverDriveSystem
       // TODO: Account for heartbeat procedure i.e. "messageCount": 132
       sscanf(
           response.c_str(),
-          R"({ "is_operational": %d, "drive_mode": "%c", "speed": %d, "angle": %d })",
-          &mc_data.is_operational, &mc_data.drive_mode, &mc_data.speed,
+          R"({ "serverHeartbeat": %d, is_operational": %d, "drive_mode": "%c", "speed": %d, "angle": %d })",
```

Comment:
This needs to match exactly what is being returned from websever JSON. This needs to be renamed to 'heartbeatCount'